Develop3

adelikade opened the pull request.

adelikade committed:
First version

adelikade committed:
add allure report

nikolay-977 commented on pom.xml:
```diff
@@ -1,17 +1,102 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0"
-         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```
Можно улучшить: хорошей практикой является добавление README.md с описанием проекта, которое включает название, технологии в проекте (например, Java 11, JUnit 4.13.2, maven 3.9.0 и т.д.), настройка (если необходима) и как запускать (mvn clean test). Как оформлять README.md можно ознакомиться в статье https://dzen.ru/a/YbzudflBPE1IDJF-

adelikade replied:
Расширила README.md  в ветке develop3

nikolay-977 commented on src/main/java/pojo/Login.java:
```diff
@@ -0,0 +1,27 @@
+package pojo;
```
Необходимо исправить наименование пакетов в соответствии с правилами нейминга. Пакеты — допустимы только строчные буквы. В случае, если слов несколько, их следует разделять точкой. Префикс должен быть одним из доменных имен верхнего уровня, таких как com, ru и т.д. Наименование пакета должно отражать суть содержимого.

adelikade replied:
Переименовала

nikolay-977 commented on src/test/java/AccountTest.java:
```diff
@@ -0,0 +1,106 @@
+    @DisplayName("Переход к Личному кабинету")
+    @Description("Проверка перехода по клику на \"Личный кабинет\"")
+    public void transferToAccount() {
+        accountPage.clickToPersonalAccountLink();
```
Необходимо исправить: Добавь проверки во все тесты, где они отсутствуют. Тест это не набор действий. Каждый тест должен что-то проверять.

adelikade replied:
В каждый тест класса LoginTest добавлена проверка перехода к конструктору после логина (метод loginPage.isDisplayedTextPutBurger());
В классе AccountTest в метод transferToAccount() добавлена проверка отображения информации о личном кабинете accountPage.isDisplayedInfoAboutAccount();
В остальных методах проверки уже были.

adelikade committed:
В каждый тест класса LoginTest добавлена проверка перехода к конструктору после логина (метод loginPage.isDisplayedTextPutBurger());
В классе AccountTest в метод transferToAccount() добавлена проверка отображения информации о личном кабинете accountPage.isDisplayedInfoAboutAccount();
Также исправлены названия пакетов и пересоздан отчет в Allure

adelikade committed:
Update README.md

Расширено описание проекта и добавлена инструкция по запуску
